packages/shared/src/utils/constants/market-types.ts

```diff
@@ -0,0 +1,6 @@
+export const MARKET_TYPES = {
+    SYNTHETIC: 'synthetic',
+    UNREGULATED: 'gaming',
```

Comment:
As a first step, having a constant file is a very good initiative 👍🏼 

We need to find an alternative for the word `UNREGULATED` to make sure the changes are future proof and it is best for us to check with Compliance team.

@shahzaib-deriv make use of #team_compliance_frontend channel on Slack to contact them and keep me in the loop.

Reply:
Sure thing, Thanks @matin-deriv 
Please let me raise in the mentioned channel to get the input from compliance team 🙏